core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -49,10 +49,12 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
     var isNationalWonder = false
     fun isAnyWonder() = isWonder || isNationalWonder
     var requiredBuilding: String? = null
+    // Deprecated
```

Comment:
Annotation?

Reply:
you can ask @yairm210, it has been replaced with "Requires a [﹉] in all the cities＂